src/main/java/net/sf/jabref/logic/formatter/casechanger/Title.java

```diff
@@ -16,6 +17,11 @@ public Title(String title) {
         this.words.addAll(new TitleParser().parse(title));
     }
 
+    public List<Word> getWordsReadOnly() {
```

Comment:
I think it will break it... See @tobiasdiez comment above. The thing is that `forEach(Word::toUpperFirst)` actually modifies the `Title` object, which it can't (I think) if `Collections.unmodifiableList(words);` is returned. On the other hand, I do not really see the use case of `Title` unless one is allowed to modify the content.